Twoliter.toml

```diff
@@ -1,5 +1,5 @@
 schema-version = 1
-release-version = "3.2.0"
+release-version = "3.2.1"
```

Comment:
Do we want this version bump in this PR?

Reply:
Nice catch. We at least want this as a separate commit, even if it lives in this PR.